cartridges/int_klarna_payments/cartridge/scripts/signin/rsaToPem.js

```diff
@@ -0,0 +1,63 @@
+var Encoding = require('dw/crypto/Encoding');
+function getRSAPublicKey(modulus_b64, exponent_b64) {
```

Comment:
A few more comments would be nice to the long term maintanance. 
* Utility functions for RSA key handling and encoding
 *
 * This module provides functionality to generate an RSA public key in PEM format
 * from its modulus and exponent, using ASN.1 DER encoding. The resulting key is 
 * formatted for use in cryptographic operations.
 *